src/data/firebase-auth.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  apiKey: "AIzaSyBklunZBwf0YMFbpHWVmfcu2scFAb2gq3o",
+  authDomain: "todomvc-ccb8e.firebaseapp.com",
+  databaseURL: "https://todomvc-ccb8e.firebaseio.com",
+};
```

Comment:
Unlike many API keys, Firebase credentials are safe to store in source control. This is because they offer no authorization themselves, they exist purely for identification purposes.